btcjson/dashevocmds.go

```diff
@@ -10,67 +10,136 @@ package btcjson
 // QuorumCmdSubCmd defines the sub command used in the quorum JSON-RPC command.
 type QuorumCmdSubCmd string
 
+// Quorum commands https://dashcore.readme.io/docs/core-api-ref-remote-procedure-calls-evo#quorum
 const (
 	// QuorumSign indicates the specified host should be added as a persistent
 	// peer.
 	QuorumSign QuorumCmdSubCmd = "sign"
 
 	// QuorumInfo indicates the specified peer should be removed.
 	QuorumInfo QuorumCmdSubCmd = "info"
+
+	// QuorumList lists all quorums
+	QuorumList QuorumCmdSubCmd = "list"
+
+	QuorumSelectQuorum QuorumCmdSubCmd = "selectquorum"
+	QuorumDKGStatus    QuorumCmdSubCmd = "dkgstatus"
+	QuorumMemberOf     QuorumCmdSubCmd = "memberof"
 )
 
+// DetailLevel is the level of detail used in dkgstatus
+type DetailLevel int
+
+// Detail Levels for dkgstatsu
+const (
+	DetailLevelCounts             DetailLevel = 0
+	DetailLevelIndexes            DetailLevel = 1
+	DetailLevelMembersProTxHashes DetailLevel = 2
+)
+
+// LLMQType is the type of quorum
 type LLMQType int
 
+// Enum of LLMQTypes
+// https://github.com/dashpay/dips/blob/master/dip-0006.md#current-llmq-types
 const (
 	LLMQType_50_60  LLMQType = 1   //every 24 blocks
-	LLMQType_400_60          = 2   //288 blocks
-	LLMQType_400_85          = 3   //576 blocks
-	LLMQType_100_67          = 4   //every 24 blocks
-	LLMQType_5_60            = 100 //24 blocks
+	LLMQType_400_60 LLMQType = 2   //288 blocks
+	LLMQType_400_85 LLMQType = 3   //576 blocks
+	LLMQType_100_67 LLMQType = 4   //every 24 blocks
+	LLMQType_5_60   LLMQType = 100 //24 blocks
 )
 
 // QuorumCmd defines the quorum JSON-RPC command.
-type QuorumSignCmd struct {
-	LLMQType    LLMQType
-	RequestId   string
-	MessageHash string
-	QuorumHash  string
-	Submit      bool
+type QuorumCmd struct {
+	SubCmd QuorumCmdSubCmd `jsonrpcusage:"\"info|list|sign\""`
```

Comment:
I think three should be more in the jsonrpcusage now right?

Reply:
I added the other 3 commands